testsuite/tests/mgc/reconciliation/test_gw_doesnt_exist.py

```diff
@@ -0,0 +1,40 @@
+"""Tests that DNSPolicy/TLSPolicy is rejected if the Gateway does not exist at all"""
+
+import pytest
+
+from testsuite.gateway import CustomReference
+from testsuite.policy.dns_policy import DNSPolicy
+from testsuite.tests.mgc.reconciliation import tls_policy
+
+pytestmark = [pytest.mark.mgc]
+
+
+@pytest.fixture(scope="module", autouse=True)
+def commit():
```

Comment:
What does this fixture do? I dont see it replacing any parent fixture. Did you mean `hub_policies_commit` from tests/mgc/conftest.py ?